README.md

```diff
@@ -25,21 +27,31 @@ pip install -e .
 
 You can now run the download and the conversion to CLDF. While downloading, insert "Y" when asked for ND-data. You do not need to download the audio files, and leaving them out will speed up the downloading. When prompted for the path to CLTS, type `../clts/` to link to the cloned CLTS-repository.
 
+TODO: `download` was very slow for me during testing (with ND data, without audio data). As I was mentioning in private communication, using VPN connection helped and solved that issue. I still can't download the files 'normally' without VPN. This is most likely not worth mentioning in this README, but it might be good to keep in mind in case someone else experiences these issues.
+
+TODO: Since CLTS is not part of the catalog arguments (unlike pylexibank etc.) and specified afterwards, would it be necessary to checkout a specific version tag of CLTS beforehand?
+
 ```
 cldfbench download cldfbench_doreco.py
 cldfbench makecldf cldfbench_doreco.py --glottolog-version v4.8
 ```
 
 The last step created a SQLite database out of the CLDF data, to quickly access all the data. Once you have the SQLite database ready, you need to install the pre-written views. Run SQL:
 
+TODO: Mention (additional) system dependencies at the beginning? E.g. `sqlite3` etc.
+
 ```
 cldf createdb cldf/Generic-metadata.json doreco.sqlite
 sqlite3 -echo doreco.sqlite < etc/views.sql
 cd ../
 ```
 
+TODO: With the added `cd ../` the following code block is potentially confusing, i.e. explicitly mention to `cd initial_lengthening/`?
```

Comment:
What is confusing about this? I add a note that on Windows, this might work differently.

Reply:
That was my bad, sorry, you can ignore that. Out of habit I never clone a repo from within another repo (i.e. I have `initial_lengthening/`, `doreco/`, and `clts/` separate from each other) so the `cd ../` would put me not into the top-level of `initial_lengthening/`. Following the instructions correctly works, since you clone inside of `initial_lengthening/`. So all good!

Reply:
We implement it in that way in most CALC papers, since it assures that people are working with the correct version. And most reviewers might not have CLTS/Concepticon etc saved locally beforehand. It's also part of the gitignore, so it doesn't get uploaded to any repo